monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/VennDiagram/ArcNode/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import * as d3 from 'd3'
+
+class ArcNode extends React.Component{
+    
+    render() {
+
+        let {prefix, ref, index, data} = this.props;;
+
+        let arc = d3.arc().innerRadius(data.inner).outerRadius(data.outer).startAngle(0).endAngle(Math.PI * 2.0);
+
+        return ( 
+            
+            <g transform={'rotate(180)'} id={data.node.pillar}>
+            <path
+            
+                id={this._reactInternalFiber.key}
```

Comment:
Why do you need to use `_reactInternalFiber` here?

Reply:
I have used the key as id to avoid multi namings, did it in another way now